ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/91521_SCAPP_apply-to-become/91489_Apply-to-become-get-application-data-map.js

```diff
@@ -0,0 +1,402 @@
+/// <reference types ='Cypress'/>
+import {data} from "../../fixtures/cath121-body.json"
+
+describe('91489: Apply-to-becom GET data application types', () => {
+    let url = Cypress.env('url')
+    let dataAppSch = data.applyingSchools[0]
+    
+    afterEach(() => {
+        cy.storeSessionData()
+    });
+    
+    before(() => {
+        cy.login()
+        cy.get('[id="school-name-44"]').click()
+        cy.visit(url+'/school-application-form/519')
+    });
+
+    after(() => {
+        cy.clearLocalStorage()
+    });
+
+    // Overview
+    it('TC01: Overview - Application Lead', () => {
+        cy.get('[test-id="Overview1_key"]').should('contain.text', 'Application to join')
+        cy.get('[test-id="Overview1_value"]').should('contain.text', data.name + ' with ' + dataAppSch.schoolName)
+        cy.get('[test-id="Overview2_key"]').should('contain.text', 'Application reference')
+        cy.get('[test-id="Overview2_value"]').should('contain.text', data.applicationId)
+        cy.get('[test-id="Overview3_key"]').should('contain.text', 'Lead applicant') 
+        cy.get('[test-id="Overview3_value"]').should('contain.text', data.applicationLeadAuthorName)
+    })
+
+    // Details of Trust
+    it('TC02: Trust details', () => {
+        cy.get('[test-id="Overview_Details1_key"]').should('contain.text', 'Trust name')
+        cy.get('[test-id="Overview_Details1_value"]').should('contain.text', data.name)
+        // 'Upload evidence' IS NOT BE COVERED IN THIS TEST AS THESE ARE UPLOAD LINKS - [test-id="Overview_Details2_key"]
```

Comment:
There is no expectation for this field to appear on the application form view at present.

Reply:
Notes added to script.